code/game/atoms_movable.dm

```diff
@@ -364,6 +364,7 @@
 	SSthrowing.processing[src] = TT
 	TT.tick()
 
+	return TRUE

```

Comment:
This isn't present on TG, what problem were ya tryin' to solve?

Reply:
/tg/ does use it, except that they're using the dot notation (https://github.com/tgstation/tgstation/blob/ad1f0fb81d53318a6c8a96e14bbb08f4716447b0/code/game/atoms_movable.dm#L566). I can't exactly remember what required the throw_at check to pass, but it was something Lavaland related.